data/applications.yaml

```diff
@@ -34,6 +34,19 @@ sources:
 supported_os: [Darwin, Windows]
 urls: ['https://github.com/mac4n6/macMRU-Parser']
 ---
+name: MicrosoftOutlookFiles
```

Comment:
Maybe make this an artifacts list and split this in
MicrosoftOutlookPABFiles
MicrosoftOutlookPSTFiles
MicrosoftOutlookOSTFiles